tests/test_psf.py

```diff
@@ -264,12 +275,28 @@ def checkPiffDeterminer(self, **kwargs):
             stars.sourceCat,
             exposure=self.exposure
         ).psfCandidates
-        psf, cellSet = self.psfDeterminer.determinePsf(
-            self.exposure,
-            psfCandidateList,
-            metadata,
-            flagKey=self.usePsfFlag
-        )
+
+        logger = logging.getLogger("lsst.psfDeterminer.Piff")
```

Comment:
Does this name come from `PiffPsfDeterminerTask._DefaultName`?  I don't think I ever knew that before...